src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -916,10 +886,10 @@ static int create_sharedmem_resources(const char* dirname, const char* filename,
     return -1;
   }
 
+#if !defined(_AIX)
   // Verify that we have enough disk space for this file.
   // We'll get random SIGBUS crashes on memory accesses if
   // we don't.
-
   for (size_t seekpos = 0; seekpos < size; seekpos += os::vm_page_size()) {
     int zero_int = 0;
     result = (int)os::seek_to_file_offset(fd, (jlong)(seekpos));
```

Comment:
os::seek_to_file_offset could, at least for the posix platforms, also be unified (separate RFE).